gulpfile.js

```diff
@@ -6,6 +6,7 @@ const header = require('gulp-header');
 const concat = require('gulp-concat');
 const replace = require('gulp-replace');
 const pump = require('pump');
+const util = require('util');
```

Comment:
Is this some kind of default package or did you forget to commit your `package.json`?

Reply:
This is&nbsp;a&nbsp;built&#x2011;in&nbsp;API: https://nodejs.org/api/util.html